index.js

```diff
@@ -61,8 +64,20 @@ const register = (server, options) => {
       return;
     }
     if (event.error && event.error.output) {
-      // some server errors emit two request events, make sure we are responding to only one:
-      if (!event.tags.includes('handler')) {
+      // make sure it has at least one of the requiredTags:
```

Comment:
https://stackoverflow.com/questions/16312528/check-if-an-array-contains-any-element-of-another-array-in-javascript using .some might be cleaner. Just create a contains helper function and call it